lib/kcerts/ssh_darwin.go

```diff
@@ -0,0 +1,32 @@
+// +build darwin
```

Comment:
`// +build linux darwin` doesn't seem to work, even though this guide says it should: https://www.digitalocean.com/community/tutorials/building-go-applications-for-different-operating-systems-and-architectures

Therefore, `ssh_darwin.go` and `ssh_linux.go` are identical except for the first line which tells the go compiler which file to select when the platform flag is provided.